packages/@ama-sdk/swagger-builder/src/core/post-processes/flatten-conflicted-allof.ts

```diff
@@ -4,8 +4,8 @@ import { PostProcess } from './post-process.interface';
 /**
  * Flatten conflicted AllOf post process
  * The purpose is to apply the composition to the definitions depending of a _ started definition
- * @example
- * // override MyCustomDefinition from @dxapi-spec/core-private to add the "example" field
+ * @example Override MyCustomDefinition from `@dxapi-spec/core-private` to add the "example" field
```

Comment:
is `@dxapi-spec/core-private` the best example here or we can find something more generic?